fundamental-concepts/data-structures-stdlib/init_background.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo initializing...
+
+wget https://katacoda.com/drsk/courses/fundamental-concepts/propose-accept-pattern/assets/create-daml-app.tar.gz
+
+daml new create-daml-app empty-skeleton
+
+tar xzf create-daml-app.tar.gz
+
+cd create-daml-app
+
+echo done
```

Comment:
We might not need this whole file at all unless we get the user to copy/paste some code etc. Have to think about it, but we could create a new DAML project, remove the standard files and create empty ones that a user would fill (as you did for the Propose/Accept pattern)

Reply:
all of it depends on whether we actually use any of this. I found it hard to come up with an interactive setting beside the quizzes for the content of this katacoda.

Reply:
we now do some copy pasting and the scenario is initialized via tar and some awk magic as usual.